brave/src/main/java/com/linecorp/armeria/client/brave/BraveClient.java

```diff
@@ -141,4 +163,9 @@ public HttpResponse execute(ClientRequestContext ctx, HttpRequest req) throws Ex
             return delegate().execute(ctx, req);
         }
     }
+
+    private void logTiming(Span span, String prefix, long startTimeMicros, long durationNanos) {
+        span.annotate(startTimeMicros, prefix + ".start");
+        span.annotate(startTimeMicros + TimeUnit.NANOSECONDS.toMicros(durationNanos), prefix + ".end");
```

Comment:
Will JVM optimize away the string concatenation here?

Reply:
Guess not - switched to hardcoding